app/views/wpcc/your_contributions/new.html.erb

```diff
@@ -4,7 +4,7 @@
   <h2 class="wpcc__heading">2. <%= t('wpcc.contributions.title') %></h2>
   <div class="contributions__row">
     <p class="t-contributions_description">
-      <%= t('wpcc.contributions.description', eligible_salary: @your_contribution.eligible_salary) %>
+      <%= t('wpcc.contributions.description', eligible_salary: number_with_delimiter(@your_contribution.eligible_salary, :delimiter => ',')) %>
```

Comment:
@tomas-stefano @margOnline - I have moved this into a presenter format... This might need some more refactoring... But as I am going on holiday I'd like to get this branch merged in... 
Could you take a look and let me know what you think of the current implementation. I might move this to it's own seperate PR if more changes are required and thus won't hold up the cucumber PR...